dev-aws/kafka-shared-msk/cbc/cbc.tf

```diff
@@ -0,0 +1,16 @@
+resource "kafka_topic" "FraudEvents" {
+  name = "cbc.FraudEvents"
+
+  replication_factor = 3
+  partitions         = 15
+
+  config = {
+    "remote.storage.enable" = "true"
+    "retention.bytes"       = "-1"
+    "retention.ms"          = "-1"
```

Comment:
Do you need infinite retention for dev?

Reply:
I'm a bit concerned that not having it be infinite could cause issues (not with existing services of course). If it is a problem for MSK we can analyse changing this value, but to be fair, we don't even have that many events in dev to begin with.

Reply:
>If it is a problem for MSK

No issues for MSK, just wanted to raise it to see if it was intentional.